docs/src/layout/page/demo/demo.jsx

```diff
@@ -1152,14 +1128,14 @@ class PageDemo extends React.PureComponent {
                                 />
                             )}
 
-                            <div
+                            {/* <div
                                 className={classes.dataGroupsContainer}
                             >
                                 <Page.DataGroups
                                     columns={mainCoulumn}
                                     data={mainData}
                                 />
-                            </div>
+                            </div> */}
```

Comment:
Is this commented out for a reason?

Reply:
Yeah, I commented it out because, we need to remove it envtually by putting it maybe in a drawer or something. I don't know. I was just trying to make the demo page look sort of nice. It's not great.

Reply:
Got it.  Can we maybe add a comment?
```JavaScript
{/* TODO: Move this to a Drawer or something.  For now, commenting it out to clean up Demo Page */}
```

Reply:
Added.

Reply:
Gracias señor!  Resolving.